### PR TITLE
Bump and Dockerfile

### PR DESCRIPTION
**Pull request by tetra12**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,41 +31,36 @@ JSON:
...
-									]
-								}
-							]
-						}
-					]
-				}
-			}
-		}
-	}
+  "apps": {
```
Comment (mholt) [inline README.md:34]: Undo these formatting changes :monocle_face:

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,10 @@
+ARG CADDY_VER=2.6.0
```
Comment (mholt) [inline Dockerfile:1]: Caddy 2.6.1 is being released currently, just FYI. The CI failed though due to timeout. We're trying to find a workaround...

(Would rather use 2.6.1 instead)

File: go.mod [hunk 3]
```diff
@@ -3,114 +3,118 @@ module github.com/mholt/caddy-grpc-web
 go 1.18
 
 require (
-	github.com/caddyserver/caddy/v2 v2.5.2
+	github.com/caddyserver/caddy/v2 v2.6.1
```
Comment (mholt) [inline go.mod:6]: Ah, good, I'm glad 2.6.1 is here at least -- that was quick!

**Review by mholt — CHANGES_REQUESTED** [review]
Thanks for the PR!

I feel like the Docker image should also use 2.6.1 as soon as it's ready. Which should be soon...

File: README.md (hunk 1 above)
Comment (tetra12) [reply README.md]: Reverted

File: Dockerfile (hunk 2 above)
Comment (tetra12) [reply Dockerfile]: Changed to 2.6.1

**Review by mholt — APPROVED** [review]
Thank you!

**Pull request merged**